shared/tests/utility/math/filter/PFVanDerPolModel.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2013 NUBots <nubots@nubots.net>
+ */
+
+#ifndef SHARED_TESTS_PFVANDERPOLMODEL_H
+#define SHARED_TESTS_PFVANDERPOLMODEL_H
+
+#include <Eigen/Core>
+
+namespace shared {
+namespace tests {
+
+    template <typename Scalar>
```

Comment:
Might be worth having a documentation comment about what this model is, maybe providing a link to describing it.